codegen/CodeGenerator/src/test/java/AggregateProcessFunctionWriterTest.java

```diff
@@ -0,0 +1,18 @@
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+public class AggregateProcessFunctionWriterTest {
+
+    @Test
+    void q6() throws Exception {
+        Node node = JsonParser.parse("C:\\Users\\Danish Ibrahim\\Desktop\\gui-codegen\\codegen\\JsonUtils\\src\\test\\resources\\Q6.json");
```

Comment:
The test is still using the absolute path.

Reply:
done